.buildkite/pipeline.yml

```diff
@@ -89,17 +89,39 @@ steps:
   - group: "🔬 Instrumented tests"
     steps:
       - label: ":wordpress: 🔬 Instrumented tests"
+        key: "WP_instrumented_tests"      
         command: ".buildkite/commands/instrumented-tests.sh wordpress"
         plugins: *common_plugins
         artifact_paths:
           - "**/build/instrumented-tests/**/*"
 
+      - label: ":wordpress: 🔍 Test Analytics"
+        depends_on:
+          - "WP_instrumented_tests"
+        command: buildkite-agent artifact download '**/test_result_1.xml' . --step "WP_instrumented_tests"
+        plugins:
+          - test-collector#v1.8.0:
+              files: "**/test_result_1.xml"
+              format: "junit"
+              api-token-env-name: "BUILDKITE_ANALYTICS_TOKEN_INSTRUMENTED_TESTS_WORDPRESS"          
```

Comment:
Not a blocker for this PR, but this plugin configuration is quite similar to the one for JP, except for the `api-token-env-name` option. It would be great if we could reuse it as we do with `*common_plugins` but I couldn't think of a way to properly handle the option variation. 🤔